content/release-10-17-0/devices-10-17-0.md

```diff
@@ -137,6 +140,22 @@ Build version</th>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>Device Management</td>
+<td>In the SmartREST template editor, issues with the presentation of the <b>External ID type</b> field for Inventory POST messages have been fixed. Under <b>CSV preview</b> the generated "Template creation CSV" has been adjusted to include the ID/externalId/externalIdType values.</td>
+<td>DM-2093</td>
+<td>10.17.0.159</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In case of bad connectivity or network delay it could happen that a gateway device went to the state of being disconnected. This resulted in operation execution being suspended. This issue is now resolved.</td>
```

Comment:
```suggestion
<td>In case of bad connectivity or network delay gateway devices could go to a state where they disconnected. This resulted in operation execution being suspended. This issue is now resolved.</td>
```

Reply:
Something is missing now. "...where they were disconnected"? At least in the original sentence they do not disconnect themselves.